packages/walletconnect/index.d.ts

```diff
@@ -1 +0,0 @@
-declare module '@trustwallet/walletconnect'
```

Comment:
I thought this file was needed to use walletconnect package, any chance to double check?

Reply:
This gets added back in a later commit. Some of the changes in these first two commits may not make 100% sense

Reply:
🚀 then